src/components/sitepages/SiteTargets.vue

```diff
@@ -249,7 +346,25 @@ export default {
 
             use_custom_date_location: false,
             skychart_date: new Date(),
-            skychart_location: [0,0]
+            skychart_location: [0,0],
+
+            show_results: false,
+            show_toggle: '',
+
+            // Directly copied from PlanTargets.vue
+            site_info: {},
+            easylist: list,
+            target: {},
+            targlist: '',
+            selected_target_obs: '',
+            lat1: '',
+            lon1: '',
```

Comment:
More descriptive variable names would help. In general, it's better to have code that's easy-to-read rather than easy-to-write.